src/validations/constraints/unit-tests/address-type-PASS.yaml

```
@@ -2,7 +2,7 @@
 test-case:
   name: Test Valid Address Type Use
   description: Test that the specified address type is valid.
-  content: ../content/ssp-all-VALID.xml # the content to run
+  content: ../../../content/rev5/examples/ssp/xml/fedramp-ssp-example.oscal.xml # the content to run
```

Non-blocking, I will approve, but having that comment one of the 200+ configuration files seems unhelpful.

```suggestion
  content: ../../../content/rev5/examples/ssp/xml/fedramp-ssp-example.oscal.xml
```